library/Rules/BIC.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Respect\Validation\Rules;
+
+use malkusch\bav\BAV;
+use Respect\Validation\Exceptions\ComponentException;
+
+/**
+ * Validates a BIC (Bank Identifier Code).
+ * 
+ * Currently only German BIC validation is supported (country code "de").
+ * The German validator depends on the composer package malkusch/bav.
+ * Note: It is not recommended to use this validator with BAV's default
+ * configuration. Use a configuration with one of the following
+ * DataBackendContainer implementations:
+ * PDODataBackendContainer or DoctrineBackendContainer.
+ * 
+ * @author Markus Malkusch <markus@malkusch.de>
+ * @see BAV::isValidBIC()
+ * @see \malkusch\bav\Configuration
+ * @see \malkusch\bav\ConfigurationRegistry::setConfiguration()
+ */
+class BIC extends AbstractRule
+{
+    /**
+     * @var String $countryCode The ISO 639-1 country code.
+     */
+    private $countryCode;
+    
+    /**
+     * Sets the country code.
+     * 
+     * The country code is not case sensitive.
+     * 
+     * @param string $countryCode The ISO 639-1 country code.
+     */
+    public function __construct($countryCode)
+    {
+        parent::__construct();
```

Comment:
You may keep it, actually it's a good practice what you did, but in our case it's unnecessary ([see the constructor yourself](https://github.com/Respect/Validation/blob/master/library/Rules/AbstractRule.php#L14-L17)), but keep it would not be a problem at all.